pkg/common/utils/resource/configmap.go

```diff
@@ -75,6 +77,7 @@ var defMap = map[string]int32{
 	BRPC_PORT:              8060,
 	BROKER_IPC_PORT:        8000,
 	BRPC_LISTEN_PORT:       5000,
+	ARROW_FLIGHT_SQL_PORT:  9090,
```

Comment:
if not configed the arrow flight is not started. operator should not config responded port.

Reply:
It has been changed to not expose the port when arrowFlightPort is -1